examples/client/qt5client.cpp

```diff
@@ -85,22 +85,23 @@ void VncViewer::paintEvent(QPaintEvent *event)
 void VncViewer::start()
 {
     cl = rfbGetClient(8, 3, 4);
-    cl->format.depth = 24;
-    cl->format.depth = 16;
-    cl->format.bitsPerPixel = 16;
-    cl->format.redShift = 11;
-    cl->format.greenShift = 5;
-    cl->format.blueShift = 0;
-    cl->format.redMax = 0x1f;
-    cl->format.greenMax = 0x3f;
-    cl->format.blueMax = 0x1f;
-    cl->appData.compressLevel = 9;
-    cl->appData.qualityLevel = 1;
-    cl->appData.encodingsString = "tight ultra";
+    cl->format.depth = 32;
+    // cl->format.depth = 16;
+    // cl->format.bitsPerPixel = 16;
+    // cl->format.redShift = 11;
+    // cl->format.greenShift = 5;
+    // cl->format.blueShift = 0;
+    // cl->format.redMax = 0x1f;
+    // cl->format.greenMax = 0x3f;
+    // cl->format.blueMax = 0x1f;
+    // cl->appData.compressLevel = 9;
+    // cl->appData.qualityLevel = 1;
+    // cl->appData.encodingsString = "tight ultra";
+    cl->appData.forceTrueColour = TRUE;
+    cl->appData.useRemoteCursor = FALSE;
     cl->FinishedFrameBufferUpdate = finishedFramebufferUpdateStatic;
     cl->serverHost = strdup(serverIp.c_str());
     cl->serverPort = serverPort;
-    cl->appData.useRemoteCursor = TRUE;
```

Comment:
Please don't move code randomly around and don't leave commented-out code. I'd like to keep the 16 bit for educational purposes - after all, this is just an example. Any other changes to improve here welcome!